client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -27,7 +27,6 @@ import { getConnectUrlForSiteId } from 'calypso/state/memberships/settings/selec
 import { useSiteGlobalStylesStatus } from 'calypso/state/sites/hooks/use-site-global-styles-status';
 import { getEnhancedTasks } from './task-helper';
 import { getLaunchpadTranslations } from './translations';
-import { type Task } from './types';
```

Comment:
Is there a reason for us to remove the import as type? Importing the type can positively affect the performance, as it is not included in the bundle.

Reply:
This task definition was duplicated with the i
import { LaunchpadInternal, Task } from '@automattic/launchpad';
I unify the definitions and remove it from the './task/

Reply:
Fixed  [4b7796c](https://github.com/Automattic/wp-calypso/pull/86432/commits/4b7796ca6abc377dbf047c76d4d4852a1c75df7b)